app/controllers/application_controller.rb

```diff
@@ -404,6 +404,10 @@ def mfa_policy
     @mfa_policy ||= MfaPolicy.new(current_user)
   end
 
+  def create_account_banner_a_b_test
```

Comment:
this is super close to the `create_an_account_banner_ab_test` method? also they return different things? that one returns the bucket, and this one seems to retunrn a boolean?

Reply:
Ah yeah, I just need one that notes whether or not the test is being run.